app/helpers/admin_helper.rb

```diff
@@ -15,4 +15,14 @@ def nav_link(text, path)
       link_to text, path, options
     end
   end
+
+  def payout_report_status_header(account_type)
+    report_date = PayoutReport.most_recent_final_report.created_at.strftime("%b %d")
```

Comment:
Nit picking: Can we do `%Y-%m-%d`?

Reply:
I agree that your format is better for most things, but in this case I wonder if it makes it less readable.  The date here should really only be from the current month if everything is working, so I think including the year might be overkill.

![image](https://user-images.githubusercontent.com/12549658/53175241-8b12d580-35b9-11e9-8a5f-3bc943a9835e.png)
vs
![image](https://user-images.githubusercontent.com/12549658/53175271-9bc34b80-35b9-11e9-8610-49578593a9b1.png)

@corymcdonald @dlipeles do you have an opinion?

Reply:
BD has been asking us to make things more user friendly so I'll vote for month/day

Reply:
I will keep it this way unless you want to veto @yachtcaptain23